src/components/ng-redux.ts

```diff
@@ -14,8 +14,10 @@ import {
 
 import { Observable } from 'rxjs/Observable';
 import { BehaviorSubject } from 'rxjs/BehaviorSubject';
+
 import 'rxjs/add/operator/map';
 import 'rxjs/add/operator/distinctUntilChanged';
+import 'rxjs/add/operator/combineLatest';
```

Comment:
I'd leave this out of ng-redux - and leave it upto the consumers to impor tthe operators that they need. internally - we need distinctUntilChanged - but not map and combineLatest